normandy/base/schema.py

```diff
@@ -17,10 +17,42 @@ class Meta:
 
 class Query(object):
     all_users = graphene.List(UserType)
+    user = graphene.Field(
+        UserType, id=graphene.Int(), username=graphene.String(), email=graphene.String()
+    )
     all_groups = graphene.List(GroupType)
+    group = graphene.Field(GroupType, id=graphene.Int(), name=graphene.String())
 
     def resolve_all_users(self, info, **kwargs):
         return User.objects.all()
 
+    def resolve_user(self, info, **kwargs):
+        id = kwargs.get("id")
+        username = kwargs.get("username")
+        email = kwargs.get("email")
+
+        if id is not None:
+            return User.objects.get(id=id)
```

Comment:
Is there some way to indicate that passing more than one of `id`, `username`, and `email` doesn't make sense?